aldryn_config.py

```diff
@@ -1,9 +1,12 @@
 # -*- coding: utf-8 -*-
+
```

Comment:
why not ``from __future__ import unicode_literals`` and get rid of all the ``u' '``'s ?